monkey/common/configuration/agent_sub_configurations.py

```diff
@@ -4,6 +4,24 @@
 
 @dataclass(frozen=True)
 class CustomPBAConfiguration:
+    """
+    Dataclass for the configuration of custom post-breach actions
+
+    Attributes:
+        linux_command (str): Command to run on Linux victim machines. If a file is uploaded,
+                             use this field to change its permissions, execute it, and/or delete
+                             it.
+                             Example: `chmod +x file.sh; ./file.sh; rm file.sh`
+        linux_filename (str): Name of the file to upload on Linux victim machines.
+                              Example: `i-am-custom-pba-file.sh`
```

Comment:
Do we need an example filename? I would expect everyone knows what a filename should look like.